CHANGES.rst

```diff
@@ -1,6 +1,19 @@
 1.7.3 (unreleased)
 ==================
 
+resample_spec_step
```

Comment:
This heading can be changed to just "resample_spec"

Reply:
Done.